src/components/UserInformationContainer/UserInformationContainer.stories.tsx

```diff
@@ -11,6 +11,7 @@ storiesOf('UserInformationContainer', module)
     <UserInformationContainer
       isSignedIn
       avatar={avatar}
+      onClickBalance={(network: Network) => console.log(network)}
```

Comment:
I think story book provides something for onThings which displays when it is clicked. Instead of relying on a console log

Reply:
In order to do this, we need to change the way we define the story. I think we should re-visit the way we instantiate components in this library and start changing them one by one.